examples/src/main/java/io/opencensus/examples/helloworld/QuickStart.java

```diff
@@ -0,0 +1,106 @@
+import java.util.Random;
+import java.util.logging.Logger;
+
+/** Simple program that collects data for video size. */
+public final class QuickStart {
+
+  private static final Logger logger = Logger.getLogger(QuickStart.class.getName());
+
+  private static final Tagger tagger = Tags.getTagger();
+  private static final ViewManager viewManager = Stats.getViewManager();
+  private static final StatsRecorder statsRecorder = Stats.getStatsRecorder();
+  private static final Tracer tracer = Tracing.getTracer();
+
+  // frontendKey allows us to break down the recorded data
+  private static final TagKey FRONTEND_KEY = TagKey.create("my.org/keys/frontend");
+
+  // videoSize will measure the size of processed videos.
+  private static final MeasureLong VIDEO_SIZE = MeasureLong.create(
+      "my.org/measure/video_size", "size of processed videos", "MBy");
+
+  // Create view to see the processed video size distribution broken down by frontend.
+  private static final View.Name VIDEO_SIZE_VIEW_NAME = View.Name.create("my.org/views/video_size");
+  private static final View VIDEO_SIZE_VIEW = View.create(
+      VIDEO_SIZE_VIEW_NAME,
+      "processed video size over time",
+      VIDEO_SIZE,
+      Aggregation.Distribution.create(
+          BucketBoundaries.create(
+              Arrays.asList(0.0, (double) (1 << 8), (double) (1 << 16)))),
+      Collections.singletonList(FRONTEND_KEY),
+      Cumulative.create());
+
+  /** Main launcher for the QuickStart example. */
+  public static void main(String[] args) throws InterruptedException {
+    TagContextBuilder tagContextBuilder =
+        tagger.currentBuilder().put(FRONTEND_KEY, TagValue.create("mobile-ios9.3.5"));
+    SpanBuilder spanBuilder =
+        tracer.spanBuilder("my.org/ProcessVideo")
+            .setRecordEvents(true)
+            .setSampler(Samplers.alwaysSample());
+    viewManager.registerView(VIDEO_SIZE_VIEW);
+    LoggingTraceExporter.register();
+
+    // Process video.
+    // Record the processed video size.
+    try (Scope scopedTags = tagContextBuilder.buildScoped();
+        Scope scopedSpan = spanBuilder.startScopedSpan()) {
+      tracer.getCurrentSpan().addAnnotation("Start processing video.");
+      // Sleep for [0,10] milliseconds to fake work.
+      Thread.sleep(new Random().nextInt(10) + 1);
+      statsRecorder.newMeasureMap().put(VIDEO_SIZE, 25648).record();
+      tracer.getCurrentSpan().addAnnotation("Finished processing video.");
+    } catch (Exception e) {
+      tracer.getCurrentSpan().addAnnotation("Exception thrown when processing video.");
+      tracer.getCurrentSpan().setStatus(Status.UNKNOWN);
+      logger.severe(e.getMessage());
+    }
+
+    // Wait for a duration longer than reporting duration (5s) to ensure spans are exported.
+    Thread.sleep(5100);
+    ViewData viewData = viewManager.getView(VIDEO_SIZE_VIEW_NAME);
+    logger.info(
+        String.format("Recorded stats for %s:\n %s", VIDEO_SIZE_VIEW_NAME.asString(), viewData));
```

Comment:
Is there a way to log the trace annotations?

Reply:
The `LoggingTraceExporter` will log `SpanData` to command line [every 5 seconds](https://github.com/census-instrumentation/opencensus-java/blob/master/impl_core/src/main/java/io/opencensus/implcore/trace/export/ExportComponentImpl.java#L29), and that should include all the information on these annotations (the message and time stamp). That's also why we need to wait 5.1s before exiting.

I added one more log to explain this, which matches [the Go example](https://github.com/census-instrumentation/opencensus-go/blob/master/examples/helloworld/main.go#L85).